### PR TITLE
Minor improve for mmcsd

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary

- mmcsd: Add gotextcsd callback to sdio_dev_s 
- mmcsd: Fix mmc card error but system can not catch it

## Testing
Pass CI

### CODE REVIEW

**Comment by pkarashchenko** [comment]
In general looks good

**Review by jerpelea — APPROVED** [review]
LGTM

File: include/nuttx/sdio.h [hunk 1]
```diff
@@ -539,6 +539,24 @@
...
+ *   buffer - Ext Csd data
+ *
+ * Returned Value:
+ *   None.
+ *
+ ****************************************************************************/
+
+#define SDIO_GOTEXTCSD(dev,buffer) \
+    ((dev)->gotextcsd?(dev)->gotextcsd(dev,buffer):OK)
+
```
[the remark this answers is not in the document]
Comment (xiaoxiang781216) [reply include/nuttx/sdio.h]: Done

**Review by pkarashchenko — APPROVED** [review]
LGTM!

**Pull request merged**